content/tutorials/esc-setup-psp-creds/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Setup Pulumi Service Provider using ESC
+title_tag: Setup Pulumi Service Provider using ESC
+layout: single
+description: Create an ESC Environment to store your Pulumi Service Provider credentials and easily re-use them across your IaC stacks.
+meta_desc: Create an ESC Environment to store your Pulumi Service Provider credentials and easily re-use them across your IaC stacks.
+meta_image: meta.png
+weight: 999
+summary: |
+    [Pulumi ESC](https://pulumi.com/esc) makes it easy to store and retrieve static and dynamic configuration settings, manage them securely and flexibly, and use them in your applications. In this tutorial, you will create an ESC environment to store credentials for the Pulumi Service Provider, then use this environment to create new Pulumi Cloud resources, including more ESC Environments.
+youll_learn:
+    - How to store and retrieve configuration values with Pulumi ESC
+    - How to configure Pulumi Service Provider
+    - How to use Pulumi Service Provider to create more ESC Environments
+prereqs:
+    - Basic understanding of Pulumi IaC the [Pulumi CLI](/docs/iac/download-install/) installed
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - Familiarity with TypeScript and Node.js
+estimated_time: 10
+collections:
+    - pulumi-esc
+---
+
+{{% notes %}}
+This tutorial is all about using ESC with Pulumi Service Provider (PSP for short!). However, using ESC is an encouragement rather than a requirement. If you *don't* want to use ESC, refer to [PSP setup docs](/registry/packages/pulumiservice/installation-configuration/) instead.
```

Comment:
personal preference but `Pulumi Service Provider (PSP)` is probably fine (don't need `for short`)

Reply:
I was trying for a bit more informal tone, since it's a tutorial, not documentation, but yeah, maybe a bit much 😄